en/docs/references/concepts/authorization/client-authentication.md

```diff
@@ -49,7 +55,7 @@ The main steps of the flow are as follows.
 The JWT **must** contain some REQUIRED claim values and **may** contain some OPTIONAL claim values. For more information on the
 required and optional claim values needed for the JWT for private_key_jwt authentication, click [here](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication).
 The authentication token **must** be sent as the value of the client_assertion parameter. The value of the client_assertion_type parameter
```

Comment:
Why do we need to have must and may in bold? Also, why are REQUIRED and OPTIONAL in title case?

Reply:
In the spec: https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication, REQUIRED and OPTIONAL is in title case I think that is why we have used title case here